tests/test_fyle/test_tasks.py

```
@@ -209,3 +210,79 @@ def test_update_non_exported_expenses(db, create_temp_workspace, mocker, api_cli
+    # First create some expenses and expense groups
+    expenses = list(data["expenses_spent_at"])
+    for expense in expenses:
+        expense['org_id'] = 'orHVw3ikkCxJ'
+        expense['employee_email'] = 'regular.user@fyle.in'  # Set a non-matching email
+
+    expense_objects = Expense.create_expense_objects(expenses, 1)
+    expense_groups = ExpenseGroup.create_expense_groups_by_report_id_fund_source(expense_objects, 1)
+
+    # Create task log and error for one expense group
+    expense_group = expense_groups[0]
+    task_log = TaskLog.objects.create(
```

Ruff (0.8.2)

Local variable `task_log` is assigned to but never used

Remove assignment to unused variable `task_log`

(F841)